README.rst

```diff
@@ -139,3 +145,107 @@ call CMake in your build directory as follows::
     cmake -D SILKIT_BUILD_DOCS=ON -B _build
     cmake --build _build --target Doxygen
 
+Platform support
+----------------------------------------
+
+SIL Kit provides three tiers of platform support
+
+.. list-table:: Support Tiers
+
+   * - Tier 1
+     - Essential targets. Continuously tested and
+       official binary packages are provided.
+   * - Tier 1
+     - Officially supported targets. Continuously tested 
+       but NO binary packages are provided
+   * - Tier 3
+     - | Targets for which we have (limited) build support but which
+       | are not continuously tested and no packages are provided from us
+
+A platform
+hereby is defined by a combination of the used operating system (OS), the CPU
+Architecture (eg. x86 or ARM64) and the compiler/toolchain used. An example
+would be `Ubuntu 20.04 x86_64 Clang 10`.
+
+SIL Kit should compile and run on
+any POSIX platform. If you have feedback for different targets or platforms
+please report via the GitHub issues, thanks!
+
+A target may be upgraded to
+`Tier 1` once we have continuous testing for it in place and we have binary
+packages available for it.
+
+
+Tier 1
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Essential targets. Automatically tested and provided as official binary packages.
+
+.. list-table:: Platform Support
+   :header-rows: 1
+
+   * - OS
+     - Architecture
+     - Notes
+   * - Windows
+     - 64bit (x86_64)
+     - MSVC 19 with Toolset 14.1
+   * - Windows
+     - 32bit (x86)
+     - MSVC 19 with Toolset 14.1
+   * - Ubuntu 18.04
+     - amd64
+     - GCC 8
+   * - Ubuntu 20.04
+     - amd64
+     - Clang 10
+
+Tier 2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Officially supported and automatically tested. NO binary packages provided
+
+.. list-table:: Platform Support
+   :header-rows: 1
+
+   * - Windows
+     - 64bit (x86_64)
+     - MSYS2/Mingw: GCC 14
```

Comment:
it';s good to mention MSYS2 here, as there are many different MinGW builds out there. Not sure if our CI workflow files are enough as documentation on how to compile, or if we need a dedicated `Building SIL Kit on different architectures` documentation in the future.

Reply:
A section on MinGW would be good, since some of the problems with MingGW/MSYS2 are which toolchain to install and use, which is not too intuitive for newcomers to the MinGW platform.